assets/vue/components/PageIncident.vue

```diff
@@ -10,7 +10,7 @@
           </li>
         </ul>
       </div>
-      <div class="smelt-link">
+      <div class="smelt-link" v-if="(incident.type ?? 'smelt') === 'smelt'">
```

Comment:
How about we generalize smelt to "request management system" so that we can put something here whenever we have something useful?

Reply:
I guess we can put something like `https://smelt.suse.de/incident/123` into the `url` field and replace "Link to Smelt" section entirely with "Link to incident" and render the url there. At this point it will be either the link to smelt or gitea.

Reply:
Yes, we could do that. I would leave cleaning up old ways of doing things to another PR and focus on what we actually want to achieve.